specification/purview/data-plane/Azure.Analytics.Purview.Insights/preview/2022-03-01-preview/purviewinsights.json

```diff
@@ -0,0 +1,1287 @@
+        "x-ms-examples": {
+          "AssetSnapshotsTags_List": {
+            "$ref": "./examples/AssetSnapshotsTags_List.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SnapshotRecord": {
+      "required": [
+        "kind"
+      ],
+      "type": "object",
+      "properties": {
+        "kind": {
+          "enum": [
+            "AssetFilesAggregation",
+            "AssetDistribution",
+            "DataEstateHealth",
+            "FilesWithoutResourceSet",
+            "AllAssetsDistribution",
+            "AssetCurationDistribution",
+            "DeletedAssetsDistribution",
+            "MostViewedAssets",
+            "NetNewAssetsDistribution",
+            "TopSearchedKeywords"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AssetType",
+            "modelAsString": true
+          }
+        },
+        "snapshotTime": {
+          "format": "date-time",
+          "type": "string"
+        }
+      },
+      "discriminator": "kind"
+    },
+    "Snapshot": {
+      "type": "object",
+      "properties": {
+        "snapshotTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SnapshotRecord"
+          }
+        }
+      }
+    },
+    "ErrorModel": {
+      "type": "object",
+      "properties": {
+        "errorCode": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "exceptionCulprit": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorModel"
+          }
+        }
+      }
+    },
+    "ErrorResponseModel": {
+      "type": "object",
+      "properties": {
+        "error": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          ]
+        }
+      }
+    },
+    "DetailsRecord": {
+      "required": [
+        "kind"
+      ],
+      "type": "object",
+      "properties": {
+        "kind": {
+          "enum": [
+            "AssetFilesAggregation",
+            "AssetDistribution",
+            "DataEstateHealth",
+            "FilesWithoutResourceSet",
+            "AllAssetsDistribution",
+            "AssetCurationDistribution",
+            "DeletedAssetsDistribution",
+            "MostViewedAssets",
+            "NetNewAssetsDistribution",
+            "TopSearchedKeywords"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AssetType",
+            "modelAsString": true
+          }
+        },
+        "snapshotTime": {
+          "format": "date-time",
+          "type": "string"
+        }
+      },
+      "discriminator": "kind"
+    },
+    "Details": {
+      "type": "object",
```

Comment:
Many type does not have description. Please at least add description for important/top-level types.

Reply:
updated